hedera-mirror-rosetta/app/services/construction/construction_service.go

```diff
@@ -225,10 +225,7 @@ func (c *ConstructionAPIService) handleCryptoTransferPayload(operations []*rType
 			return nil, errors.Errors[errors.InvalidAccount]
 		}
 
-		amount, err := parse.ToInt64(operation.Amount.Value)
-		if err != nil {
-			return nil, errors.Errors[errors.InvalidAmount]
-		}
+		amount, _ := parse.ToInt64(operation.Amount.Value)
```

Comment:
Why would you ignore the error if `operation.Amount.Value` cannot be parsed to `int`?

Reply:
This check is performed at 214 in validator.ValidateOperationsSum(operations), so I removed it in order not to double check what is already done.

Reply:
what if a bug is introduced in `ValidateOperationsSum` or the logic is changed there so that it does not perform that check, 2 months from now.

Would the one changing the logic see that you are ignoring the error here and add `err` check? I doubt it